model/gamut_compress.ctl

```diff
@@ -2,17 +2,25 @@
 // <ACESuserName>ACES 1.3 Look - Gamut Compress</ACESuserName>
 
 //
-// "LMT" for compressing out of gamut scene-referred values into AP1.
-// 
-// Input and output are ACES2065-1.
+// Gamut compressor to bring out of gamut scene-referred values into AP1
 //
 
+//
+// Usage:
+//  The intended use of this transform is immediately after the IDT, so that all grading
+//  or compositing operations are downstream of the compression, and therefore work only
+//  with positive AP1 values. It is however not recommended to bake the compression into
+//  VFX pulls, as it may be beneficial for compositors to have access to the unmodified
+//  image data.
 //
 // Direction:
-//    By default this transform operates in the forward direction (compressing the gamut).
-//    If instead an inverse operation is needed (undoing a prior gamut compression), there
-//    is a runtime flag to do this. In ctlrender, this can be achieved by appending 
-//    '-param1 invert 1' after the '-ctl gamut_compress.ctl' string.
+//  By default this transform operates in the forward direction (compressing the gamut).
+//  If instead an inverse operation is needed (undoing a prior gamut compression), there
+//  is a runtime flag to do this. In ctlrender, this can be achieved by appending 
```

Comment:
> In ctlrender, this can be achieved by appending

Do we need passive tense here?

> In ctlrender, this is achieved by appending

Reply:
This wording is taken directly from the [Rec.709 ODT CTL](https://github.com/ampas/aces-dev/blob/518c27f577e99cdecfddf2ebcfaa53444b1f9343/transforms/ctl/odt/rec709/ODT.Academy.Rec709_100nits_dim.ctl#L32).